catalog/content/endpoint/content-endpoint-rest/src/main/java/ddf/content/endpoint/rest/ContentEndpoint.java

```diff
@@ -408,7 +408,7 @@ protected Response doCreate(InputStream stream, String contentType, String direc
                     responseBuilder = Response
                             .ok(byteArrayInputStream, createResponse.getCreatedMetadataMimeType());
                 } else {
-                    responseBuilder = Response.ok();
+                    responseBuilder = Response.ok("{}");
```

Comment:
The jQuery File Upload Plugin expects json and wasn't able to parse an empty 200 response.